docs/development/metrics.md

```diff
@@ -0,0 +1,36 @@
+# Monitoring
+
+etcd-druid uses [Prometheus][prometheus] for metrics reporting. The metrics can be used for real-time monitoring and debugging of compaction jobs.
+
+The simplest way to see the available metrics is to cURL the metrics endpoint `/metrics`. The format is described [here](http://prometheus.io/docs/instrumenting/exposition_formats/).
+
+Follow the [Prometheus getting started doc][prometheus-getting-started] to spin up a Prometheus server to collect etcd metrics.
+
+The naming of metrics follows the suggested [Prometheus best practices][prometheus-naming]. All compaction related metrics are put under namespace `etcddruid` and subsystem `compaction`.
+
+### Compaction
+
+These metrics give an idea about the compaction jobs that run after some interval in shoot control planes. Studying the metrices, we can deduce how many compaction job ran successfully, how many failed, how many delta events compacted etc.
+
+| Name | Description | Type |
+|------|-------------|------|
+| jobs_total | Total number of compaction jobs initiated by compaction controller. | Counter |
```

Comment:
```suggestion
| etcddruid_compaction_jobs_total | Total number of compaction jobs initiated by compaction controller. | Counter |
```

Please specify the full name of the metrics in this table, for the below metrics as well, as it becomes easy for consumption by end-users.